### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by joaobrittoneto**
@doudou 
Made some debug and tests in the tank. 
So far so good. 
How can I test an operation? Is possible to do it in rock-display?

### CODE REVIEW

File: scripts/dock.rb [hunk 1]
```diff
@@ -3,26 +3,34 @@
 include Orocos
 Orocos.initialize
 Orocos.run 'usbl_Dock' do
+           #'auv_uwmodem_commands::UWModemDock' => 'modem_dock',
+           #'usbl_evologics::UsblDock' => 'usbl_dock' do #, valgrind: true do
+           
    
-    usbl_dock = Orocos.name_service.get 'usbl_dock'
-    modem_dock = Orocos.name_service.get 'modem_dock'
+    usbl_dock = TaskContext.get 'usbl_dock'
```
Comment (doudou) [inline scripts/dock.rb]: Orocos.get is the right one (yes, I know, it's messy)

**Comment by doudou** [comment]
Hey. Get rid of the deployments and use the task_model_name => task_name syntax instead, please.

**Comment by doudou** [comment]
For operations, best would be to use the unit tests. @rafaelsaback could help you there. Otherwise, rock-display cannot help, but there's oroshell. If you do

~~~
oroshell task_name:t
~~~

you get a Ruby REPL where 't' is the task object for 'task_name'

**Comment by doudou** [comment]
Hey:
 - try to avoid "minor change" as a commit message
 - do not put commits called "for debug purpose" on a PR. Push them to a separate branch
 - what's the point of displaying the same thing twice ??? (I refer to all these `cout <<` followed by `RTT::Log`)

**Comment by joaobrittoneto** [comment]
Sorry. 
I'll make a clean up version.

**Comment by joaobrittoneto** [comment]
@doudou 
Clean the code and remove the couts. 
Compare byte of message with letters.

**Comment by doudou** [comment]
> Sorry. 

No need to, that's the whole point of doing code reviews.

:+1: feel free to merge

**Pull request merged**